FiveCalls/FiveCalls/Welcome.swift

```diff
@@ -9,112 +9,107 @@
-                .foregroundStyle(.fivecallsDarkBlueText)
-            Text(subMessage)
-                .font(.headline)
-                .textCase(.uppercase)
-                .foregroundStyle(.fivecallsDarkBlueText)
-                .opacity(subMessageOpacity)
-                .animation(.easeIn, value: subMessageOpacity)
-            Spacer()
-            Button(action: {
-                onContinue?()
-                dismiss()
-            }) {
-                Text(R.string.localizable.welcomePage2ButtonTitle())
-                    .fontWeight(.semibold)
-                    .foregroundColor(.white)
-                    .font(.system(size: 30))
-            }
-            .padding()
-            .frame(maxWidth: .infinity, minHeight: 73)
-            .background {
-                RoundedRectangle(cornerRadius: 6)
-                    .foregroundColor(Color.fivecallsDarkBlue)
+        ScrollView {
+            Grid(verticalSpacing: 30) {
+                Image(decorative: R.image.fivecallsLogotype)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 292)
+                    .padding(.vertical, 24)
+                GridRow() {
+                    Image(systemName: "phone.badge.waveform")
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(.red, .blue)
+                        .font(.title)
+                        .accessibilityHidden(true)
+                    VStack(alignment: .leading) {
+                        Text(R.string.localizable.welcomeSection1Title())
+                            .fontWeight(.heavy)
+                        Text(R.string.localizable.welcomeSection1Message())
+                    }
+                    .accessibilityElement(children: .combine)
+                }
+                GridRow() {
+                    Image(systemName: "goforward.5")
+                        .foregroundStyle(.blue)
+                        .font(.title)
+                        .accessibilityHidden(true)
+                    VStack(alignment: .leading) {
+                        Text(R.string.localizable.welcomeSection2Title())
+                            .fontWeight(.heavy)
+                        Text(R.string.localizable.welcomeSection2Message())
+                    }
+                    .accessibilityElement(children: .combine)
+                }
+                GridRow() {
+                    Image(systemName: "person.2")
+                        .opacity(subMessageOpacity)
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(.blue, .red)
+                        .font(.title)
+                        .accessibilityHidden(true)
+                    Text(subMessage)
+                        .fontWeight(.heavy)
+                        .opacity(subMessageOpacity)
+                        .gridColumnAlignment(.leading)
+                }
+                Spacer()
```

Comment:
Do we need two spacers here?

Reply:
For the welcome view to accommodate dynamic text, I wrapped it in a ScrollView. That changes how spacers work, and two spacers gives us good space / drops the button almost to the bottom with default text sizes and a modern phone. This is also why the button gets cut off a bit on the small phone.